lib/pymedphys/_dicom/header.py

```diff
@@ -168,7 +168,7 @@ def patient_ids_in_datasets_are_equal(
         True if Patient IDs match for all datasets, False otherwise.
     """
 
-    if not len(datasets) >= 2:
-        raise ValueError("At least two datasets must be provided for comparison")
+    if len(datasets) == 1:
+        return True
```

Comment:
Nit: Is this condition needed here at all? I certainly prefer to avoid cases/if statements if they're not needed (more code branches means more differing scenarios). Not much of in issue, but I highly suspect that the statement below will work just fine for a sequence of length 1.

Reply:
Removed :)